cmd/app/options.go

```diff
@@ -363,59 +369,83 @@ func (o *Options) assignSelfhosted(envs []string) {
 		}
 	}
 
-	regexActions := map[*regexp.Regexp]func(matches []string, value string){
```

Comment:
Go maps iterate in random order, so by using a slice to ensure consistency, adding the two problematic options/regex's at the top/start of the slice